test/libsolidity/syntaxTests/iceRegressionTests/const_struct_with_mapping.sol

```diff
@@ -0,0 +1,10 @@
+contract C {
+    struct S {
+        mapping(uint => uint) c;
+    }
+    S public constant e = 0x1212121212121212121212121212121212121212;
+}
+// ----
+// TypeError 6280: (71-135): Types in storage containing (nested) mappings cannot be assigned to.
```

Comment:
Perhaps one way to fix this would be to convert this regular type error into a fatal type error.

Reply:
The crash happens even without assignment, `S public constant e;` is sufficient. And it seems that `solAssert` happens before error reporting. I'll have a closer look, thanks!